doc/geany.txt

```diff
@@ -1785,6 +1786,25 @@ This works basically the same as on other platforms::
     "c:\program files\geany\bin\geany" -g c:\mytags.php.tags c:\code\somefile.php
 
 
+Generating tags files using ctags
+*********************************
+Geany supports loading tag files generated using the ``ctags`` command-line
+tool from the universal-ctags project (https://github.com/universal-ctags/ctags).
+Even though Geany should work with any ctags file, it is recommended to use
+certain fields to give Geany some additional information. The recommended fields
+are ``EfiklsZSt``, so, for instance, to generate tags for all sources in the geany
```

Comment:
delete "for instance" and comma after "directory"

Reply:
But there should still be comma after "so", right?

I will also renamed the "geany" directory to something like "my_project" to avoid possible confusion with the geany binary.

Reply:
Could be either way, but my preference would be comma before so only.